src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -132,7 +132,7 @@
     "staged": ["RequestPriority"],
     "acquired": ["RequestPriority", "SiteWhitelist", "SiteBlacklist"],
     "running-open": ["RequestPriority", "SiteWhitelist", "SiteBlacklist"],
-    "running-closed": ["RequestPriority"],
+    "running-closed": ["RequestPriority", "SiteWhitelist", "SiteBlacklist"],
```

Comment:
I would rather not allow this site update for running-closed, as it has no effect in the work in running state in the agent at all.

Reply:
My concern with this change is that changing the site lists is not only a change of job classads, but it can potentially also affect data locality. So I'd rather keep jobs already materialized targeting the "original" sites, it likely gives higher chance to have them finding the needed input files.

Reply:
2 cents : being able to change the DesiredSites in condor or post-assignment was a very useful feature of unified-equalizor that allowed to complete workflows, that could otherwise be "stuck"